### ISSUE TITLE
Nothing is rendered

### ISSUE THREAD

**Issue by KatGr** [airbnb/react-sketchapp#63]
I am  reporting an issue.

I used this the Example Code from the API at 
Artboard. I used the example and added a new folder in the examples folders. 
Unfortunately it doesn't render a thing :(

Code:

```
import React from 'react';
import { render, Artboard } from 'react-sketchapp';


const Document = props =>
<Artboard
  name='My Artboard'
  style={{
    width: 480
  }}
>
  <Text>Hello world!</Text>
</Artboard>

export default (context) => {
  render(<Document />, context.document.currentPage());
}
```

Sketch version:
43.1
A Screenshot:
![screen shot 2017-05-02 at 13 42 00](https://cloud.githubusercontent.com/assets/26592218/25616460/25860b3a-2f3d-11e7-9eae-ad5ac9d4076b.png)
![screen shot 2017-05-02 at 13 51 05](https://cloud.githubusercontent.com/assets/26592218/25616677/7a86e5b8-2f3e-11e7-91ba-8823d1a8f308.png)

**Comment by jongold** [comment]
@KatGr a few things happening here:
- if you're using skpm, you shouldn't need to set up babel or webpack! anything you add to `webpack.config.js` will override skpm's config, so you can just add the extra loaders (and you don't need json-loader since webpack@2 :))
- I don't think style-loader / css will work right now (though I'd love to be proved wrong!) - see discussion in #29 

I'd suggest commenting out your custom config until it starts working again, then carefull add things back in

**Comment by KatGr** [comment]
I completly deleted the webpack config.
It still doesn't show anything :(

![screen shot 2017-05-03 at 08 52 01](https://cloud.githubusercontent.com/assets/26592218/25650525/ce2c5cee-2fdd-11e7-8abd-542b354ea033.png)

**Comment by zmdavis** [comment]
I found that having a `.babelrc` file will also result in nothing being rendered into Sketch.

**Comment by jongold** [comment]
@KatGr weird :( . can you upload a .zip of your folder?

@zmdavis that's to be expected - it will override [`skpm`'s default config](https://github.com/sketch-pm/skpm/blob/master/lib/utils/webpackConfig.js#L8-L30). We're using `babel-preset-airbnb` & `babel-plugin-add-module-exports` - feel free to use a different config at your own risk!

**Comment by zmdavis** [comment]
Yep, not a problem -- I just thought it might be why @KatGr is still having trouble.  From her first screenshot it looks like she added a `.babelrc` file.

**Comment by KatGr** [comment]
I deleted the .babelrc but it also didn't work. So i deleted the whole thing and cloned it again. No example runs on npm run render:
![screen shot 2017-05-04 at 10 13 35](https://cloud.githubusercontent.com/assets/26592218/25695183/66637ee6-30b2-11e7-9c17-87e93cb05731.png)

But the npm run link-plugin works fine  o.o

But now I need some loaders (Even if I am not sure if they are going to work), how can I add them as writing a completely new webpack.config.js wasn't the right way?

**Comment by mathieudutour** [comment]
there is a space in your path `WYSIWYG - Projekt`, that's why is failing. @jongold, it's coming from https://github.com/jongold/run-sketch-plugin/blob/master/src/index.js#L12, probably need to wrap `${coscript}` in double quotes or something

**Comment by jongold** [comment]
Sorry for the delay!
🐞  I published run-sketch-plugin@1.0.2 — if you try deleting your node_modules directory and reinstalling it should work

re: loaders — which ones?
